bin/tauri.conf.json

```diff
@@ -4,6 +4,7 @@
   "build": {
     "beforeDevCommand": "yarn workspace @ethui/gui dev",
     "beforeBuildCommand": "yarn workspace @ethui/gui build",
+    "beforeBundleCommand": "./scripts/postbuild.sh",
```

Comment:
@DavideSilva did you validate if this works with the release action?

Reply:
I didn't run the action itself, no. I wasn't sure if that would mess up or trigger an actual releases. But this should work with the action since the only thing the action is doing is running `tauri build` and this change happens during that command execution, similar to the `beforeBuildCommand` that we already had